datastream-server-api/src/main/java/com/linkedin/datastream/server/api/strategy/AssignmentStrategy.java

```diff
@@ -66,4 +66,21 @@
       Set<DatastreamTask>> currentAssignment, DatastreamGroupPartitionsMetadata datastreamPartitions) {
     return currentAssignment;
   }
+
+  /**
+   * Move a partition for a datastream group according to the targetAssignment.
+   * It returns a map from instance -> tasks map with partition info stored in the task.
+   *
+   * This interface needs to be implemented if the Brooklin Coordinator is going to perform the
+   * partition movement.
+   *
+   * @param currentAssignment the old assignment
+   * @param targetAssignment the target assignment retrieved from Zookeeper
```

Comment:
Since currentAssignment and targetAssignment has same naming convention it is slightly confusing that both of them are going to be a Map of host -> datastreamTasks. but targetAssignment is a map of host -> topic Partitions  right? Can you please make that clear through the documentation. 
Can you also make it clear that movePartitions can perform movePartitions a single datastream Group and but multiple move calls across different hosts?

Reply:
updated, thanks for the suggestion.